File: PhysicalTherapy/PhyscialTherapy/Models/Routine.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace PhysicalTherapy.Models
{
    public class Routine
    {
        public string Description { get; set; }

        public DateTime Date { get; set; }

        public bool IsComplete { get; set; }

        public bool IsNew { get; set; }

        public ICollection<MessageLog> ListOfMessageLogs { get; set; }

        public string Name { get; set; }

        public Patient Patient { get; set; }

        public int PatientId { get; set; }

        public ICollection<RoutineExercise> RoutineExercises { get; set; }

        public int RoutineId { get; set; }

        public ICollection<PostRoutineSurvey> postRoutineSurveys { get; set; }
    }
}

File: PhysicalTherapy/PhysicalTherapy/Client/src/app/models/routine.ts
import { MessageLog } from "./MessageLog";
import { Patient } from "./Patient";
import { PostRoutineSurvey } from "./PostRoutineSurvey"
import { RoutineExercise } from "./RoutineExercise";

export interface Routine {
    date : Date;
    description : string,
    isComplete : boolean,
    isNew : boolean,
    listOfMessageLogs : MessageLog[],
    name : string;
    patient : Patient
    patientId : number,
    routineExercises : RoutineExercise[],
    routineId : number,
    postRoutineSurvey : PostRoutineSurvey,
    postRoutineSurveyId : number
}